### ISSUE TITLE
Support for Rootless Pachyderm Deployment for OpenShift

### ISSUE THREAD

**Issue by mindthevirt** [pachyderm/pachyderm#6262]
<div> Freshdesk Ticket ID: <a href="https://pachyderm-support.freshdesk.com/a/tickets/256">256</a></div>
                       <div> Freshdesk Ticket Agent: Dale Georg</div>
                       <div> Freshdesk Ticket Agent Email: dale@pachyderm.io</div>
                       <div> Ticket Priority: Medium</div>
                       <div> Freshdesk Ticket Description: </div><div style="font-family:-apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica Neue, Arial, sans-serif; font-size:14px"><div dir="ltr">This is a tracking issue for the work to be completed to support non-root running of Pachyderm. This includes modifications to core Pachyderm and the Pachyderm Helm chart.</div></div>

**Comment by mindthevirt** [comment]
<div>Agent Dale Georg linked Freshdesk ticket <a href="https://pachyderm-support.freshdesk.com/a/tickets/256">256</a> for this issue.</div>

**Comment by chainlink** [comment]
Related: https://github.com/pachyderm/pachyderm/issues/6643

**Comment by mindthevirt** [comment]
<div>Agent David Echo linked Freshdesk ticket <a href="https://pachyderm-support.freshdesk.com/a/tickets/309">309</a> for this issue.</div>

**Comment by mindthevirt** [comment]
<div>Agent Dale Georg unlinked Freshdesk ticket <a href="https://pachyderm-support.freshdesk.com/a/tickets/309">309</a> from this issue.</div>

**Comment by brendoncarroll** [comment]
Yeah all of our containers will run as 1000.  User containers will run as whatever they set with User or in the image.
